### PR TITLE
Added Income Model 

### PR DESCRIPTION
**Pull request by joshuaseetss**
Fixed Checkstyle

### CODE REVIEW

File: src/main/java/seedu/address/model/income/Income.java [hunk 1]
```diff
@@ -0,0 +1,130 @@
...
+    private final Phone phone;
+    private final Description description;
+
+    //Data fields
+    private final Amount amount;
+    private final Set<Tag> tags = new HashSet<>();
+    private final Date dateAdded;
+
+    /**
+     * Every field must be present and not null.
```
Comment (lawncegoh) [inline src/main/java/seedu/address/model/income/Income.java]: Javadocs should be uniform with a present tense verb starting the javadoc

File: src/main/java/seedu/address/model/income/Income.java [hunk 2]
```diff
@@ -0,0 +1,130 @@
...
+     * This defines a weaker notion of equality between two incomes.
+     */
+    public boolean isSameIncome(Income otherIncome) {
+        if (otherIncome == this) {
+            return true;
+        }
+
+        return otherIncome != null
+                && otherIncome.getName().equals(getName())
+                && (otherIncome.getDescription().equals(getDescription()) || otherIncome.getPhone().equals(getPhone()));
```
Comment (lawncegoh) [inline src/main/java/seedu/address/model/income/Income.java]: this line might be a little too long, can split into various variables possible?

**Comment by lawncegoh** [comment]
Good job! time for integration

File: src/main/java/seedu/address/model/income/Income.java [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+import java.util.Set;
+
+import seedu.address.model.claim.Amount;
+import seedu.address.model.claim.Description;
+import seedu.address.model.commonvariables.Name;
+import seedu.address.model.commonvariables.Phone;
+import seedu.address.model.tag.Tag;
+
+
+
```
Comment (weigenie) [inline src/main/java/seedu/address/model/income/Income.java]: Perhaps you would like to remove these additional blank lines?

Comment (joshuaseetss) [reply src/main/java/seedu/address/model/income/Income.java]: Great suggestion! Thank you

File: src/main/java/seedu/address/model/income/Income.java (hunk 1 above)
Comment (joshuaseetss) [reply src/main/java/seedu/address/model/income/Income.java]: Oh right, thank you!

**Pull request merged**